scrapers/TmwVRnet.yml

```diff
@@ -0,0 +1,60 @@
+name: "TMWVRNET"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - tmwvrnet.com
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - tmwvrnet.com
+    scraper: movieScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $info: &infoSel //div[@class="content__main"]
+      $about: &aboutSel //div[@class="about-video"]
+    scene:
+      Title: &titleSel $about//h1[@class="title"]/text()
+      Date: &dateAttr
+        selector: $info//img[@alt="Release date"]/following-sibling::div/text()
+        postProcess:
+          - parseDate: January 02, 2006
```

Comment:
It should be
```yaml
          - parseDate: January 2, 2006
```
as the site doesnt use zero padding for the day

Reply:
Yup, you are absolutely right. Can't believe I missed this one!